frontend/lib/about/about_page.dart

```diff
@@ -105,6 +106,13 @@ class AboutPage extends StatelessWidget {
                             builder: (context) => LicensePage(),
                           ));
                     }),
+                ListTile(
+                    leading: Icon(Icons.code_outlined),
+                    title: Text("Quellcode dieser App auf github.com"),
```

Comment:
```suggestion
                    title: Text("Quellcode der App"),
```

I think we dont need to mention github here